deebot_client/capabilities.py

```diff
@@ -118,6 +124,8 @@ class CapabilityCleanAction:
 class CapabilityClean:
     """Capabilities for clean."""
 
+    auto_empty_enable: CapabilitySetEnable[AutoEmptyEnableEvent] | None = None
+    auto_empty: CapabilitySetTypes[AutoEmptyModeEvent, AutoEmptyMode] | None = None
```

Comment:
It can be one capability as a robot always supports both or none in my opinion

Reply:
Did i understand you correct, i make here a change:
https://github.com/MVladislav/client.py/commit/f4a4d4d22aff6ef1fce7507d48a5907f23e222b4